quick-guide-for-principal-engineers.md

```diff
@@ -0,0 +1,68 @@
+
+# The Way We Work: Quick Guide for Principal Engineers
+
+## Your Role
+
+Your role is to use software engineering best practices to support rapid and safe delivery of high-quality software.
+
+## Your Focus
+
+At this stage in your career you should focus on the success and professional development of
+
+- your direct reports (i.e. the people who you line manage)
+- your indirect reports (i.e. the people who your direct reports line manage)
+- the people operating within multiple (possibly all) Delivery Teams
+
+## Your Responsibilities
+
+Your key responsibilities are to:
+
+- Ensure our work complies with **Engineering Quality Framework** standards and best practices
+- Undertake **Strategy Implementation and Lean Improvement Projects**
+- Carry our deliberate **Coaching** to help others adapt and grow
+- Undertake your **Line Management** duties, in accordance with guidance
+- Make **Payments to Contractors**, ensuring controls are in place
+
+When the above responsibilities are met, you may also:
+
+- Lead **System Design** activities within a Delivery Team
+- Ensure there is sufficient **Documentation** to support our products and systems
+- Attend SDMT to assist with **SDMT Operational Planning**
+- Carry out deliberate **Mentoring** to help others develop their knowledge and skills
+
+Details on how these responsibilities should be carried out can be found in The Way We Work.
+
+## How You Will Be Supported
+
+Your line manager will:
```

Comment:
Line manager: A line manager should:

or

Line manager: Your line manager is expected to:

or

Your line manager is required to: